main.py

```diff
@@ -1,10 +1,66 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_race(race_info: dict) -> None:
+    if not Race.objects.filter(
+            name=race_info["name"]
+    ).exists():
```

Comment:
You can skip splitting into lines here, i suppose. You'll have the same code style throughout the code. Fix it in the next function as well